server/apps/microservices/src/processors/asset-uploaded.processor.ts

```diff
@@ -40,7 +40,9 @@ export class AssetUploadedProcessor {
 
     // Video Conversion
     if (asset.type == AssetType.VIDEO) {
-      await this.videoConversionQueue.add(JobName.MP4_CONVERSION, { asset });
+      if (asset.mimeType == 'video/quicktime') {
```

Comment:
In my opinion we shouldn't be making these kind of decisions here. The processing of only quicktime files right now is something we are looking to address, which will be done by the video conversion queue. The video conversion queue should just receive all video files and it can make the decision on whether the file actually needs converting or not.

Reply:
How should it make that decision?

Reply:
Currently the transcoder seems to transcode every file that is not mp4.

Reply:
I think we're saying we can still choose to not convert videos, but "skipping" should happen in one place and that place should be the mp4 processor implementation itself.

Reply:
I have removed the change here, but I'm wondering how should we handle iOS as it doesn't support webm direct playback apparently?